AdminUi/apps/admin_ui/lib/pages/login_page.dart

```diff
@@ -34,56 +38,128 @@ class _LoginScreenState extends State<LoginScreen> {
+          child: Align(
+            alignment: Alignment.topCenter,
+            child: SizedBox(
+              width: 500,
+              height: 250,
+              child: Card(
+                child: SizedBox(
+                  width: 300,
+                  child: Padding(
+                    padding: const EdgeInsets.only(top: 20, left: 25, right: 25),
+                    child: Column(
+                      mainAxisSize: MainAxisSize.min,
+                      children: [
+                        TextField(
+                          controller: _apiKeyController,
+                          focusNode: _apiKeyFocusNode,
+                          decoration: InputDecoration(
+                            labelText: 'API Key',
+                            border: const OutlineInputBorder(),
+                            errorBorder: const OutlineInputBorder(
+                              borderSide: BorderSide(color: Colors.red, width: 2),
+                            ),
+                            focusedErrorBorder: const OutlineInputBorder(
+                              borderSide: BorderSide(color: Colors.red, width: 2),
+                            ),
+                            errorText: (_attemptedLogin && !apiKeyValid) ? 'Invalid API Key' : null,
+                          ),
+                          onSubmitted: (_) => _login(),
+                          onChanged: (text) {
+                            setState(() {
+                              _isButtonEnabled = text.isNotEmpty;
+                              _attemptedLogin = false;
+                              apiKeyValid = false;
+                            });
+                          },
+                          obscureText: true,
+                        ),
+                        const SizedBox(height: 20),
+                        SizedBox(
+                          width: double.infinity,
+                          height: 40,
+                          child: ElevatedButton(
+                            onPressed: _isButtonEnabled ? _login : null,
+                            style: _isButtonEnabled
+                                ? const ButtonStyle(
+                                    backgroundColor: MaterialStatePropertyAll(
+                                      Color.fromARGB(255, 0, 47, 114),
+                                    ),
+                                  )
+                                : null,
+                            child: Text(
+                              'Login',
+                              style: TextStyle(color: _isButtonEnabled ? Colors.white : null),
+                            ),
+                          ),
+                        ),
+                        const SizedBox(height: 8),
+                        TextButton(
+                          onPressed: () {
+                            // TODO(stamenione): Implement what should happen when "Forgot API Key?" is pressed
```

Comment:
We don't want todos in our code. If it's unclear what the button does, then you shouldn't add it.
What good is a button with no functionality?

Reply:
right, please don't just implement what's in the design. I don't think we will have this button in the near future, so we can just ditch it until we need it.

Reply:
Unused feature removed.